ANSWERS.md

```diff
@@ -0,0 +1,17 @@
+## Self-Study/Essay Questions
+
+Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
+
+1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+
```

Comment:
Good answers Angela, all I would add to this answer is that `describe(name, fn)` blocks group together/organize _related_ tests 👍